index.html

```diff
@@ -367,8 +371,10 @@ <h4>DataIntegrityProof</h4>
           </p>
           <p>
 The `proofValue` property of the proof MUST be a detached EdDSA
-produced according to [[RFC8032]], encoded according to [[MULTIBASE]] using
-the base58-btc base encoding.
+produced according to [[RFC8032]], encoded using the base-58-btc header and
```

Comment:
The base-58-btc header alphabet is not defined.

Reply:
It's defined here: https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/pull/196.html#multibase-0